src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function getSum(number) {
+    return function(value) {
+      count++;
+
+      const result = value + number;
+
+      if (count % 2 === 0 && count > 3) {
+        return 'Bzzz... Error!';
+      } else {
+        return result;
+      }
+    };
+  }
```

Comment:
The inner function returned by `getSum` should not be named. You can simply return an anonymous function.